docs/time/index.rst

```diff
@@ -148,6 +148,16 @@ You can also use time-based `~astropy.units.Quantity` for time arithmetic:
   >>> Time("2020-01-01") + 5 * u.day
   <Time object: scale='utc' format='iso' value=2020-01-06 00:00:00.000>
 
+As of Astropy version 5.1, |Time| objects can also be passed directly to
+`numpy.linspace` to create even-sampled time arrays, including support for
+non-scalar ``start`` and/or ``stop`` points - given compatible shapes.
+
+  >>> np.linspace(t, t+[4, 120] * u.day, 4, endpoint=False)
```

Comment:
```suggestion
  >>> np.linspace(t, t + [4, 120] * u.day, 4, endpoint=False)
```

Reply:
Small point, but ideally we give an example with actual `Time` instances for both start and stop.